File: 1.1 mario_less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int rows;
    do
    {
        rows = get_int("Rows: ");
    }
    while (rows < 1 || rows > 8);

    // Cria linhas
    for (int i = 1; i <= rows; i++)
    {
        // Cria espaços em branco
        for (int j = rows; j > i; j--)
        {
            printf(" ");
        }

        // Cria colunas
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: 6.1-mario_less/mario.py
from cs50 import get_int


def set_height():
    height = get_int("Height: ")

    if height < 1 or height > 8:
        height = set_height()

    return height


height = set_height()

for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print()
